Invaders/Assets/Scripts/InvadersGame.cs

```diff
@@ -145,7 +145,7 @@ private void Awake()
         else
         {
             //We do a check for the client side value upon instantiating the class (should be zero)
-            Debug.LogFormat("Client side we started with a timer value of {0}", m_TimeRemaining);
+            //Debug.LogFormat("Client side we started with a timer value of {0}", m_TimeRemaining);
```

Comment:
Any debug lines that are commented out should be in an if statement to turn them off or on instead. I would consider using `BrainCloud.Plugin.Interface.EnableLogging` as a flag to determine if these debug lines should get printed or not.